packages/runtime/test-runtime-utils/src/assertionShortCodesMap.ts

```diff
@@ -1373,7 +1373,7 @@ export const shortCodeMap = {
 	"0x795": "Data store runtime entryPoint does not have request",
 	"0x796": "Data store runtime entryPoint does not have request",
 	"0x797": "forest must be empty",
-	"0x799": "Conflicting TreeSchema names",
+	"0x799": "Conflicting TreeNodeSchema names",
```

Comment:
Are you allowed to modify these errors directly? I guess it should be fine if the comment inlined at the callsite to `assert` is also updated. Or maybe the replace step handles it either way?

Reply:
This file is updated/regenerated from scratch with every time you run the assert policy fix script (which is usally before each release branch is cut). If that script was run now, it would make this change, so I see no reason not to include it now (and avoid the message here being out of date), but it doesn't really matter either way.